test/plain-list/AutosuggestApp.test.js

```diff
@@ -229,6 +479,25 @@ describe('Plain list Autosuggest', () => {
     });
   });
 
+  describe('when pressing Enter and selectFirstSuggestion is true', () => {
+    before(() => {
+      setSelectFirstSuggestion(true);
+    });
+
+    after(() => {
+      setSelectFirstSuggestion(false);
+    });
+
+    beforeEach(() => {
+      focusAndSetInputValue('p');
+    });
+
+    it('should hide suggestions since the first suggestion is focused', () => {
+      clickEnter();
+      expectSuggestions([]);
+    });
```

Comment:
Do you really feel like this test adds value given we have [this test](https://github.com/moroshko/react-autosuggest/blob/master/test/plain-list/AutosuggestApp.test.js#L220-L224)? I personally feel it is unnecessary.